src/core/seq.js

```
@@ -0,0 +1,124 @@
+    step = 1;
+    last = nums[1];
+  } else {
+    first = nums[0];
+    step = nums[1];
+    last = nums[2];
+  }
+  // error: zero increment
+  if (step === 0) {
+    const stderr = `seq: invalid Zero increment value: ‘${operands[1] || '0'}’\n` +
+                   "Try 'seq --help' for more information.\n";
+    return { stdout: '', stderr, env: Object.assign({}, env, { '?': 1 }) };
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)